packages/vscode/src/integrations/editor/diff-changes-content-provider.ts

```diff
@@ -5,6 +5,7 @@ interface DiffChangesData {
   filepath: string;
   content: string;
   cwd: string;
+  type: "base" | "updated";
```

Comment:
```suggestion
  type: "original" | "modified";
```
this naming may more align with vscode